_data/simple-icons.json

```diff
@@ -8956,6 +8956,12 @@
             "source": "https://litecoin-foundation.org/litecoin-branding-guidelines/",
             "guidelines": "https://litecoin-foundation.org/litecoin-branding-guidelines/"
         },
+        {
+            "title": "Literal",
+            "hex": "000000",
+            "source": "https://literal.club/pages/press",
+            "guidelines": "https://literal.club/pages/press"
```

Comment:
```suggestion
```
There aren't any guidelines on this page from what I can tell, which is fine. It isn't a required property.

Reply:
It was the closed thing that I could find so I added it.

Reply:
Here's a good example of what you should put there from a PR I just merged: https://github.com/simple-icons/simple-icons/pull/11297.